src/components/CreateTest.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import Dropdown from "./form/Dropdown";
+import RadioButtons from './form/RadioButtons'
+import apiClient from '../services/ApiClient';
+import { useState } from 'react';
+
+const CreateTest = () => {
+
+	//locations, source, comparison are hardcoded for now, ideally 
+	// we should get them from DB 
+	const locations = [
+    { label: 'us-east-2', value: 'us-east-2' },
```

Comment:
Would it make sense to collapse these two separate properties into one? I see that they are used in the Dropdown component but maybe still could use one property?

Reply:
totally! Done :)